web/src/components/filings-calendar/FilingsCalendarAsList.tsx

```diff
@@ -30,15 +29,19 @@ export const FilingsCalendarAsList = (props: Props): ReactElement => {
     setNumberOfVisibleCalendarEntries(LIST_VIEW_MORE_INCREMENT);
   }, [props.activeYear]);
 
-  const licenseCalendarEvents = getLicenseCalendarEvents(
-    props.business?.licenseData,
-    Number.parseInt(props.activeYear)
-  );
+  // const licenseCalendarEvents =
```

Comment:
Okay, so we don't have to worry about renewal events because we're removing them from the calendar. Can we add a `TODO` to make sure this gets back in the calendar?